i18n/translations/en.json

```
@@ -349,6 +349,8 @@
 	"license.manage": "Manage your licenses",
 	"license.register.help": "You received your license code after the purchase via email. Please copy and paste it to register.",
 	"license.register.label": "Please enter your license code",
+	"license.register.domain": "Register your license for <strong>{host}</strong>",
+	"license.register.local": "You are about to register your license for your local domain <strong>{host}</strong>. If this site will be deployed to a public domain, please register it there instead. If <strong>{host}</strong> is the domain you want to license Kirby to, please continue.",
```

I think it's enough to bold the domain once:

```suggestion
	"license.register.local": "You are about to register your license for your local domain <strong>{host}</strong>. If this site will be deployed to a public domain, please register it there instead. If {host} is the domain you want to license Kirby to, please continue.",
```